template.yaml

```diff
@@ -1186,6 +1227,118 @@ Resources:
             Period: 60
             Stat: Maximum
 
+  ELB4XXAnomalyDetector:
```

Comment:
Are we using these alarms or the core alarms (and are we standardising this across kiwi/orange/lime?) See https://gds.slack.com/archives/C05FSAFFD2L/p1722848605833599

Reply:
Do you have any suggestion on this @louiseli1 ? Core doesn't have load balancer in the frontend and hence they take a different approach, whereas KIWI, LIME, ORANGE use a load balancer.